csi/server/plugin/opensds/plugin.go

```diff
@@ -16,32 +16,33 @@ package opensds
 
 import (
 	"github.com/golang/glog"
-	opensdsClient "github.com/opensds/nbp/client/opensds"
+	"github.com/opensds/nbp/client/opensds"
 	"github.com/opensds/nbp/csi/server/plugin"
 	"github.com/opensds/opensds/client"
 )
 
-const (
-	// PluginName setting
-	PluginName      = "csi-opensdsplugin"
-	FakeIQN         = "fakeIqn"
-	TopologyZoneKey = "topology." + PluginName + "/zone"
-)
-
 // Plugin define
 type Plugin struct {
-	Client *client.Client
+	PluginStorageType string
+	Client            *client.Client
+	VolumeClient      *Volume
+	FileShareClient   *FileShare
 }
 
-func NewServer(endpoint, authStrategy string) (plugin.Service, error) {
+func NewServer(endpoint, authStrategy, storageType string) (plugin.Service, error) {
```

Comment:
I don't know why storageType is required to start a new server, please elaborate.

Reply:
Done.